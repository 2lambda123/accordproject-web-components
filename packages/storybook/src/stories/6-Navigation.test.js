import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  storyKindRegex: /Navigation/,
  suite: 'Navigation Storyshots'
});
