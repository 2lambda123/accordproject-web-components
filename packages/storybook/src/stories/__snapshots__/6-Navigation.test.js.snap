// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation Storyshots Navigation Navigation 1`] = `
<div
  className="sc-fKgIGh kIuOko ui-components__navigation-wrapper"
>
  <a
    className="sc-bCwgka sc-iwaifL Lllle eesuxp ui-components__navigation-switch-title-active"
    onClick={[Function]}
  >
    NAVIGATION
  </a>
  <a
    className="sc-bCwgka sc-cxNIbT Lllle jqhJAJ ui-components__navigation-switch-title-inactive"
    onClick={[Function]}
  >
    FILES
  </a>
  <div
    className="sc-lmgQde enZzfF"
  >
    <div
      className="sc-iJCRLp sc-bYwzba cFghiT cXQmeY ui-components__navigation-header-clause"
      onClick={[Function]}
    >
      This is a clause heading
    </div>
    <div
      className="sc-iJCRLp cFghiT ui-components__navigation-header-one"
      onClick={[Function]}
    >
      This is a H1 heading
    </div>
    <div
      className="sc-iJCRLp sc-giAqnE cFghiT rcNdD ui-components__navigation-header-two"
      onClick={[Function]}
    >
      This is a H2 heading
    </div>
    <div
      className="sc-iJCRLp sc-ezzayL cFghiT eeBFp ui-components__navigation-header-three"
      onClick={[Function]}
    >
      This is a H3 heading
    </div>
  </div>
</div>
`;
